Interfaces/API/Dirac.py

```diff
@@ -1008,7 +1008,7 @@ def __printOutput( self, fd = None, message = '' ):
   #     directory = directory[:-1]
   #
   #   if printOutput:
-  #     for fileKey, metaDict in listing['Value']['Successful'][directory]['Files'].items():
+  #     for fileKey, metaDict in listing['Value']['Successful'][directory]['Files'].iteritems():
```

Comment:
I see here this change in the comment. Did you replace items -> iteritems automaticaly, or did you evaluate every time whether it can be done or not ? Because this can bite us as well.